### ISSUE TITLE
Switchresx SHA256 Mismatch

### ISSUE THREAD

**Issue by Ol3xS** [Homebrew/homebrew-cask#97283]
#### Description of issue

Hello,
I cannot install switchresx cask. Any fix?

#### Command that failed

```
brew install --cask switchresx
```

#### Output of command 

```
==> Downloading https://www.madrau.com/data/switchresx/SwitchResX4.zip
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 33b2bec0623f1edd83e32efa2ac1980f9d677211abdfd8a3d62289eb29abf20a
  Actual: 3ce4b56aa95320add92ec9cb0fcc5308ad333e761fd73bb97d20cc9ee30b47e4
    File: /Users/****/Library/Caches/Homebrew/downloads/efb04f7763e301af96b96836fdbc5e3f1131dc840656b1f921de1a57848acd54--SwitchResX4.zip
To retry an incomplete download, remove the file above.
```

**Comment by adithyasunil26** [comment]
I am also getting the same error but with a different sha256 checksum as actual 

`
==> Downloading https://www.madrau.com/data/switchresx/SwitchResX4.zip
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 33b2bec0623f1edd83e32efa2ac1980f9d677211abdfd8a3d62289eb29abf20a
  Actual: 637ab376d4144fc12fed7f36affe9ea43a0c5ef0742051df92cdcc61a51577ff
    File: /Users/****/Library/Caches/Homebrew/downloads/efb04f7763e301af96b96836fdbc5e3f1131dc840656b1f921de1a57848acd54--SwitchResX4.zip
To retry an incomplete download, remove the file above.`

**Comment by Axemasta** [comment]
I am seeing the same
```
$brew install --cask switchresx
==> Downloading https://www.madrau.com/data/switchresx/SwitchResX4.zip
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 33b2bec0623f1edd83e32efa2ac1980f9d677211abdfd8a3d62289eb29abf20a
  Actual: 637ab376d4144fc12fed7f36affe9ea43a0c5ef0742051df92cdcc61a51577ff
    File: /Users/****/Library/Caches/Homebrew/downloads/efb04f7763e301af96b96836fdbc5e3f1131dc840656b1f921de1a57848acd54--SwitchResX4.zip
To retry an incomplete download, remove the file above.
```

Deleting the file & trying again leads to the same error

**Comment by adithyasunil26** [comment]
There is a change in the installation files and hence the change in the checksum. They have changed the name of the application to "SwitchResX Installer" instead of "SwitchResX". 
Fixed this issue in #97378
